elastic/kibana
# [ILM] Remove frozen phase

## Summary

After https://github.com/elastic/elasticsearch/pull/62403 was merged, this PR removes the corresponding ILM frozen phase UI.

## How to test

1. Start Kibana on basic license
2. Go to index lifecycle management
3. Create a policy with the available form controls for all available phases
4. The form should work as usual, except there should be no option to activate or configure anything regarding a frozen phase.
 
## Additional context

* Undoes work introduced here: https://github.com/elastic/kibana/pull/75968
* Happens after https://github.com/elastic/kibana/pull/76126 so code grew slightly
